test/e2e/validation_test.go

```diff
@@ -61,11 +61,13 @@ var _ = Describe("Validation", func() {
 	It("generated communication matrix should be equal to documented communication matrix", func() {
 		By("generate documented commatrix file path")
 		docType := "aws"
```

Comment:
can we change the logic here entirely so we wont assume aws as default?
https://github.com/openshift/api/blob/a463917701ab4506585893739e576ffd1e6b924e/config/v1/types_infrastructure.go#L410
we need to cover all the cases here, where we should error for platforms we dont support.

so this should be a simple switch case, which defults to not supported